mlflow/entities/model_registry/registered_model.py

```diff
@@ -94,3 +94,23 @@ def to_proto(self):
         rmd.tags.extend([ProtoRegisteredModelTag(key=key, value=value)
                         for key, value in self._tags.items()])
         return rmd
+
+    def to_dictionary(self):
+        """
+        Converts mlflow.entities.model_registry.registered_model.RegisteredModel
+        contents into a python dictionary
+        """
+        model_version_dict = {
+            "creation_timestamp": self.creation_timestamp,
+            "current_stage": self.current_stage,
```

Comment:
@emarkou Some of these fields (e.g., `current_stage` and `version`) are defined on the ModelVersion identity, yet your `to_dictionary()` method appears to be defined on the RegisteredModel entity. Did you mean to define `to_dictionary()` on ModelVersion instead? 

Also, can you provide more information about the intended use case here?

Reply:
thanks @dbczumar for the comments, I have updated the pr. Regarding the use case, when I call, for example, the `get_registered_model() ` method, I usually want to iterate over the model versions included in the `latest_versions` property, extract the information and save them in a format that would allow further manipulation. The most suitable structure for this seems to be a dictionary, exactly like mlflow already does for runs.(https://www.mlflow.org/docs/latest/_modules/mlflow/entities/run.html#Run)